Samples/TdLib.Samples.GetChats/Program.cs

```diff
@@ -20,6 +25,12 @@ internal static class Program
 
     private static async Task Main()
     {
+        // TdLib creates database in the current directory.
+        // Creating separate directory and switch to that dir.
+        var clientDb = Path.Combine(AppContext.BaseDirectory, "db");
+        Directory.CreateDirectory(clientDb);
+        Directory.SetCurrentDirectory(clientDb);
```

Comment:
A better solution: just provide `DatabaseDirectory` and `FilesDirectory` config parameters when SetTdlibParameters called.

Reply:
That’s for new version. Samples which works only from master branch is disservice to the new users. That require knowledge of the changes. I’m not sure about your release cycle so wanna play safe

Reply:
hmm... I think there is a misunderstanding between us. DatabaseDirectory and FilesDirectory params already exist, not only in master but also in NuGet versions (even older versions). We just need to provide them here: https://github.com/egramtel/tdsharp/blob/master/Samples/TdLib.Samples.GetChats/Program.cs#L101

Reply:
Oops. My bad. Let me update sample then

Reply:
Done.